src/insanely_fast_whisper/cli.py

```diff
@@ -75,35 +75,22 @@
 def main():
     args = parser.parse_args()
 
-    if args.flash == True:
-        pipe = pipeline(
-            "automatic-speech-recognition",
-            model=args.model_name,
-            torch_dtype=torch.float16,
-            device=f"cuda:{args.device_id}",
-            model_kwargs={"use_flash_attention_2": True},
-        )
-    else:
-        pipe = pipeline(
-            "automatic-speech-recognition",
-            model=args.model_name,
-            torch_dtype=torch.float16,
-            device=f"cuda:{args.device_id}",
-        )
+    pipe = pipeline(
+        "automatic-speech-recognition",
+        model=args.model_name,
+        torch_dtype=torch.float16,
+        device="mps" if args.device_id == "mps" else f"cuda:{args.device_id}",
+        model_kwargs={"use_flash_attention_2": args.flash},
+    )
 
+    if args.device_id == "mps":
+        torch.mps.empty_cache()
+    elif not args.flash:
```

Comment:
Did you try mps with bettertransformer by any chance? As far as I understand BT currently uses sdpa, but I don't remember if PyTorch supports sdpa on mps. (Just curious, I can try later).

Reply:
Yeah! didn't do much for me either. Could you loop through it to run the same transcription multiple times? THis would make sure that the model is warmed up okay.

Reply:
it if is fine to run this even when the device is mps I'd say let's do it, it will make the code a bit easier to understand
(or we could add a comment why we don't do it on mps or when flash is enabled)

Reply:
I ran a couple times looking at the transcription time and memory use (in activity monitor) and didn't notice much difference. I'd say we just leave it out as it is right now in the PR until we have a better understanding.

Reply:
LGTM! lovely!!